python/mxnet/symbol.py

```diff
@@ -35,8 +35,52 @@ def __init__(self, handle):
         self.handle = handle
 
     def __add__(self, other):
+        """add with broadcast
+
+        Examples
+        --------
+        >>> a = mx.sym.Variable('a')
+        >>> b = mx.sym.Variable('b')
+        >>> c = a + b
+
+        >>> dev = mx.cpu();
+        >>> x = c.bind(dev, args={'a': mx.nd.ones((2,2)), 'b' : mx.nd.ones((2,2))})
```

Comment:
can you do it as simple binary op, so that you can also use this for imperative?